shell/platform/linux/fl_dart_project.cc

```diff
@@ -42,6 +43,7 @@ static void fl_dart_project_dispose(GObject* object) {
   g_clear_pointer(&self->aot_library_path, g_free);
   g_clear_pointer(&self->assets_path, g_free);
   g_clear_pointer(&self->icu_data_path, g_free);
+  g_ptr_array_unref(self->dart_entrypoint_args);
```

Comment:
Use `g_clear_object(&self->dart_entrypoint_args, g_ptr_array_unref)` to ensure this can't be double freed.